commit f2c32a882d2c1cde6fc552a5a3d34b4c1330edb8
Author: Mark Brown <broonie@opensource.wolfsonmicro.com>
Date:   Sun Jun 24 12:09:45 2012 +0100

    Extcon: Arizona: Add driver for Wolfson Arizona class devices
    
    Most Wolfson Arizona class audio hub CODECs include a flexible ultra low
    power accessory detection subsystem. This driver exposes initial support
    for this subsystem via the Extcon framework, implementing support for
    ultra low power detection of headphone and headset with the ability to
    detect the polarity of a headset.
    
    The functionality of the devices is much richer and more flexible than
    the current driver, future patches will extend the features of the
    driver to more fully exploit this.
    
    Signed-off-by: Mark Brown <broonie@opensource.wolfsonmicro.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/extcon/Kconfig b/drivers/extcon/Kconfig
index 29c5cf852efc..bb385ac629a8 100644
--- a/drivers/extcon/Kconfig
+++ b/drivers/extcon/Kconfig
@@ -29,4 +29,12 @@ config EXTCON_MAX8997
 	  Maxim MAX8997 PMIC. The MAX8997 MUIC is a USB port accessory
 	  detector and switch.
 
+config EXTCON_ARIZONA
+	tristate "Wolfson Arizona EXTCON support"
+	depends on MFD_ARIZONA
+	help
+	  Say Y here to enable support for external accessory detection
+	  with Wolfson Arizona devices. These are audio CODECs with
+	  advanced audio accessory detection support.
+
 endif # MULTISTATE_SWITCH